rails_event_store_active_record/lib/rails_event_store_active_record.rb

```diff
@@ -1,6 +1,8 @@
 require 'rails_event_store_active_record/generators/migration_generator'
 require 'rails_event_store_active_record/generators/v1_v2_migration_generator'
 require 'rails_event_store_active_record/event'
+require 'rails_event_store_active_record/mappers/yaml'
+require 'rails_event_store_active_record/mappers/yaml_with_event_class_remapping'
```

Comment:
it's added just as example, we can maintain it on private code base side, but was neccessary to show reasoning of implementation (to keep flexibility for future use cases)

Reply:
If you would like to merge it, I will provide necessary specs to cover it